server/bin/create_test_repos.py

```diff
@@ -0,0 +1,469 @@
+#!/usr/bin/env python
+
+"""
+This script is used for generating testing RPM repositories
+"""
+
+import sys
+import os
+import json
+import string
+import subprocess
+import shutil
+import tempfile
+import requests
+
+
+RPM_SPEC_FILE_TEMPLATE = """
+Summary:        ${Name} package
+Name:           ${name}
+Version:        ${version}
+Release:        ${release}
+License:        GPL License
+Packager:       Jiri Hnidek <jhnidek redhat com>
+Vendor:         Red Hat
+Group:          Networking/Daemons
+URL:            http://www.tricky-testing-animals.org/
+BuildArch:      noarch
+ 
+%%description
+This is a ${name} package.
+ 
+%%files
+ 
+%%changelog
+* Fri Mar 1 2019 Jiri Hnidek <jhnidek redhat com> 0
+- ${name}
+"""
+
+GPG_NAME_REAL = "candlepin"
+
+GPG_NAME_EMAIL = "noreply@candlepinproject.org"
+
+GPG_BATCH_GEN_SCRIPT_CONTENT = """
+Key-Type: 1
+Key-Length: 2048
```

Comment:
Please use 4096 bit keys.  We're already seeing failures in places now that JVMs/operating systems are becoming more adamant about using longer keys.

Reply:
OK, np.